CHANGELOG.md

```diff
@@ -23,12 +23,14 @@
 - (#190) Possibility to `test` or `deploy` a service from a git or GitHub url
 - (#233) Add logs in the `service test` command with service logs by default and all dependencies logs with the `--full-logs` flag
 - (#235) Add `ListServices` and `GetService` APIs
+- (#) Add .mesgignore to excluding file from the Docker build
```

Comment:
This should not be places in this part of the release but in the unreleased part because the v1.0.0-beta has been released already